src/nashEquilibria.jl

```diff
@@ -0,0 +1,207 @@
+
+- `g::NormalFormGame` : A NormalFormGame object.
+
+##### Returns
+
+- `g::NormalFormGame` : A NormalFormGame object.
+"""
+function unDominate!(g::NormalFormGame)
+    changeFlag = true
+    changesMade = false
+    while changeFlag == true
+        changeFlag = unDominate_Auxiliary!(g::NormalFormGame)
+        changesMade = changeFlag
+    end
+    return changesMade
+end
+
+function unDominate_Auxiliary!(g::NormalFormGame)
+    # Iterate over players, flagging strictly dominated strategies, and then eliminating.
+    changesMade = false
+    for i in 1:num_players(g)
+        player = g.players[i]
+        numActions = num_actions(player)
+        payoffMatrix = player.payoff_array
+        dominatedActions = Array(Integer,0)
+        for j in 1:numActions
+            currentAction = payoffMatrix[j,:]
+            for k in symdiff(numActions,j)
+                 otherAction = payoffMatrix[k,:]
+                 if size(otherAction.>=currentAction) == size(currentAction)
+                      g.players[i].payoff_array = g.players[i].payoff_array[symdiff(numActions,j),:]
+                      changesMade = true
+                      return changesMade
+                 end
+            end
+        end
+    end
+end
+
+# Auxiliary function for Lemke-Howson.
+function constructTableaux(g::NormalFormGame)
+    # Check params.
+    if num_players(g) != 2
+         error("Two-player game required.")
+    end
+
+    # Bring in primitives.
+    A = g.players[1].payoff_array
+    B = g.players[2].payoff_array
+    m = num_actions(g.players[1])
+    n = num_actions(g.players[2])
+
+    # Create tableaux.
+    tableaux = Array(Matrix,2)
+    tableaux[1] = cat(2, B, eye(n), ones(n, 1))
+    tableaux[2] = cat(2, eye(m), A, ones(m, 1))
+
+    return tableaux
+end
+
+# Testing implementing using another package. Need to look at syntax for passing in maxPivots.
+
+"""
+Apply the Lemke-Howson algorithm to a normal form game.
+
+##### Arguments
+
+- `g::NormalFormGame` : A NormalFormGame object.
+- `maxPivots::Integer` : A bound on the maximum number of iterations.
+
+##### Returns
+
+- `nashEquilibrium::Tuple` : A Nash equilibrium for the game, given as a tuple of MixedAction objects.
+
+"""
+function applyLH(g::NormalFormGame, maxPivots::Integer=50000)
+     # Pre-process.
+     unDominate!(g)
+     unDegenerate!(g)
+     tableaux = constructTableaux(g)
+
+     # Solve.
+     nashEquilibrium = nonneg_lsq(tableaux[1],tableaux[2];alg=:pivot)
+     return nashEquilibrium
+end
+
+"""
+Return an ordered list of support size tuples (x, y), according to the metric introduced in this paper: http://robotics.stanford.edu/%7Eshoham/www%20papers/GEB%20computing-nash.pdf.
+
+##### Arguments
+
+- `g::NormalFormGame` : A 2-player NormalFormGame object.
+
+##### Returns
+
+- `sizeProfiles:::Array{Tuple}` : An array of integer tuples.
+"""
+function generateProfileList(g::NormalFormGame)
+    if num_players(g) != 2
+        error("Inappropriate number of players.")
```

Comment:
make this error more descriptive. Specify the number of players that were supplied and what the correct number ought to be